pkg/controller/vsphere/actuator.go

```diff
@@ -67,6 +68,13 @@ func (a *Actuator) Create(ctx context.Context, machine *machinev1.Machine) error
 		return a.handleMachineError(machine, err, createEventAction)
 	}
 	if err := newReconciler(scope).create(); err != nil {
+		metrics.RegisterFailedInstanceCreate(&metrics.MachineLabels{
```

Comment:
I'm not a fan of this metric.  It's not clear how we'll use it and it's going to be incredibly spammy.

What we need is a properly defined user story that describes what we're trying to find out, and then determine the best way to collect that information.

Reply:
I added instead more granular reporting on 
- https://github.com/openshift/machine-api-operator/pull/590/files#diff-11b6610f82e5d47fdfb8dda44b90dd1eR77-R86 - task was finished with error
and
- https://github.com/openshift/machine-api-operator/pull/590/files#diff-11b6610f82e5d47fdfb8dda44b90dd1eR536-R543 - error on starting `create machine` task

Reply:
Yeah, I think this is much better way of doing it.  I still prefer the code live in the machine-controller rather than the actuator, but I think this is okay for now as long as we're not building alert rules on top.